_generated/ArchiveFilesV2/make.json

```diff
@@ -3,7 +3,7 @@
         "archivePackages": [
             {
                 "archiveName": "7zip.zip",
-                "url": "https://vstsagenttools.blob.core.windows.net/tools/7zip/4/7zip.zip",
+                "url": "https://vstsagenttools.blob.core.windows.net/tools/7zip/5/7zip.zip",
```

Comment:
Do I understand correctly that you are upgrading the major version of `7zip`? Are there any breaking changes? How would it be tested to detect any possible regression?

Reply:
Yes, it's major version upgrading.
It is tested by different archive file formats on windows.